docker-compose/os.yaml

```diff
@@ -38,6 +40,23 @@ services:
       - 9600:9600
     profiles:
       - opensearch
+    healthcheck:
+      test: curl -u admin:admin -s -f opensearch:9200/_cat/health > /dev/null || exit 1
+      interval: 30s
+      timeout: 10s
+      retries: 5
+
+  opensearch-configurer:
```

Comment:
Shouldn't we mount the json files?

Reply:
What do you think of this setup?
I added `config/opensearch-configurer/*` and mounted them into the container. I tested the setup on my own machine using files from https://github.com/trifork/cheetah-infrastructure-utils-opensearch/tree/main/src/opensearch/tests/tc_collect (renamed by removing the namespace and configmap parts of the names)

Reply:
Yes it looks great!

I  have created a separate jira issue of having good sample json files in the `config/opensearch-configurer` directory